packages/idealo.design/src/ui/pages/BlogPage/DetailView.jsx

```diff
@@ -76,7 +76,7 @@ const BlogDetailView = (props) => {
     }
 
     if(blogpost.email !== null){
-        emailLink = <a href={blogpost.email}><img alt="" src="https://img.icons8.com/doodle/48/000000/email-sign.png"/></a>
+        emailLink = <a href={'/blog'}><img alt="" src="https://img.icons8.com/doodle/48/000000/email-sign.png"/></a>
```

Comment:
Nope, sorry, that's not the correct way to fix this. You need to prepend the email value (coming from the database) with a 'mailto:', i.e. `<a href={'mailto:'+blogpost.email}>...</a>`, then it will automatically open the email software, the user has set as their default. Also, please shortly check if the email (coming from the database) is set up correctly (e.g. `roadrunner@acme.com`).

Reply:
Thank you @nforgerit. It's in progress ;-)